src/bin/rawst.rs

```diff
@@ -1,9 +1,43 @@
-use rawst::cli::parser::init;
+use rawst::cli::args;
+use rawst::cli::args::Command;
+use rawst::cli::parser as not_the_parser;
+use rawst::core::config::Config;
 use rawst::core::errors::RawstErr;
+use rawst::core::io::config_exist;
 
 #[tokio::main]
 async fn main() -> Result<(), RawstErr> {
```

Comment:
Don't merge. This is broken.

I'm manually testing again and it seems that in this rewrite I messed up something. It should be a minor thing, but I'm debugging it right now.
I'll update the comment once I fix it.

Reply:
I had swapped the if for downloading files/list, so I was trying to run the wrong one.